tests/Unit/Evolution/Systems/Cce/Test_KleinGordonWorldtubeData.cpp

```diff
@@ -29,8 +29,10 @@ namespace {
 // dr/dt = A * f * cos ft * R
 //       =  r / (1 + A * sin ft) * A * f * cos ft
 void create_fake_time_varying_klein_gordon_modal_data(
```

Comment:
Looks good, just remove the `modal` part from this function name when squashing.